examples/obs-csi-plugin/kubernetes/bucket/pvc.yaml

```diff
@@ -0,0 +1,11 @@
+kind: PersistentVolumeClaim
```

Comment:
pls add an example of capacity expansion.

Reply:
done